### PR TITLE
Whyka add

### PR DESCRIPTION
**Pull request by nhandl3**

### CODE REVIEW

**Comment by tpvlaxmi** [comment]
@whyka Do you have any favorite restaurant in your local place.

**Comment by githubteacher** [comment]
@whyka Any restaurant suggestions?

**Comment by mbmccollum** [comment]
don't know where you are or your favorite food  but love beer and pizza

File: whyka-menu.md [hunk 1]
```diff
@@ -0,0 +1 @@
+eating
```
Comment (rsc2015) [inline whyka-menu.md:1]: just checking, how to comment?

Comment (githubteacher) [inline whyka-menu.md:1]: Just an example of Reviews.

Comment (githubteacher) [inline whyka-menu.md:1]: Another comment!

**Review by githubteacher — COMMENTED** [review]
Just showing how reviewing works

**Pull request merged**